packages/opentelemetry-tracing/test/Tracer.test.ts

```diff
@@ -220,4 +222,60 @@ describe('Tracer', () => {
     assert.strictEqual(context.traceFlags, TraceFlags.NONE);
     span.end();
   });
+
+  it('should start an active span with name and function args', () => {
+    const tracer = new Tracer(
+      { name: 'default', version: '0.0.1' },
+      { sampler: new TestSampler() },
+      tracerProvider
+    );
+
+    assert.strictEqual(tracer.startActiveSpan('my-span', span => {
+      try {
+        // assert.strictEqual(getSpan(context.active()), span)
+        return 1
+      } finally {
+        span.end();
+      }
+    }), 1);
+  });
+
+  it('should start an active span with name, options and function args', () => {
+    const tracer = new Tracer(
+      { name: 'default', version: '0.0.1' },
+      { sampler: new TestSampler() },
+      tracerProvider
+    );
+
+    assert.strictEqual(tracer.startActiveSpan('my-span', {}, span => {
+      try {
+        // assert.strictEqual(getSpan(context.active()), span)
+        return 1
+      } finally {
+        span.end();
+      }
+    }), 1);
+  });
+
+  it('should start an active span with name, options, context and function args', () => {
+    const tracer = new Tracer(
+      { name: 'default', version: '0.0.1' },
+      { sampler: new TestSampler() },
+      tracerProvider
+    );
+
+    const ctxKey = createContextKey('foo');
+
+    const ctx = context.active().setValue(ctxKey, 'bar')
+
+    assert.strictEqual(tracer.startActiveSpan('my-span', {}, ctx, span => {
+      try {
+        // assert.strictEqual(getSpan(context.active()), span)
```

Comment:
Why commented?

Reply:
todo comment added, currently `getSpan(context.active())` returns `undefined`

Reply:
Why is it todo? Is there some reason it doesn't work currently?